ci-scripts/rhdh-setup/create_resource.sh

```diff
@@ -374,6 +402,8 @@ get_token() {
     sleep 5s
   done
 
+  echo "AFTER"; cat "$token_field"
+
```

Comment:
@jhutar I assume you have this here for debugging purposes... please remove it, unless you have a specific reason to keep it here.